internal/commands/init.go

```diff
@@ -42,30 +42,31 @@ func InitConfigFile(path string) error {
 		{
 			Prompt: &survey.Input{
 				Message: " Env:",
-				Default: os.Getenv("ENV"),
+				Default: os.Getenv("IZE_ENV"),
 			},
 			Validate: survey.Required,
 			Name:     "env",
 		},
 		{
 			Prompt: &survey.Input{
 				Message: " aws region:",
-				Default: os.Getenv("AWS_REGION"),
+				Default: os.Getenv("IZE_AWS_REGION"),
 			},
 			Validate: survey.Required,
 			Name:     "aws_region",
 		},
 		{
 			Prompt: &survey.Input{
 				Message: " aws profile:",
-				Default: os.Getenv("AWS_PROFILE"),
+				Default: os.Getenv("IZE_AWS_PROFILE"),
```

Comment:
Let's have both as defaults, first check `IZE_AWS_PROFILE` and the `AWS_PROFILE`?